graphics/Coin/Portfile

```diff
@@ -1,80 +1,65 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           qt4 1.0
-PortGroup           bitbucket 1.0
+PortSystem         1.0
 
-bitbucket.setup     Coin3D coin 3.1.3 Coin-
-name                Coin
-revision            7
-license             BSD
-description         cross platform C++ OpenGL scene graph library
-long_description    Coin is a high-level 3D graphics library with a   \
-                    C++ Application Programming Interface. Coin uses scene-graph data \
-                    structures to render real-time graphics suitable for mostly all   \
-                    kinds of scientific and engineering visualization applications.
+PortGroup          cmake 1.1
+PortGroup          github 1.0
 
-conflicts           Coin-framework
-categories          graphics devel
-platforms           darwin
-maintainers         nomaintainer
+name               Coin
```

Comment:
I *think* you have to do this after the `github.setup` line, but am not 100% sure...

Reply:
Correct, this must occur _after_ `github.setup`, as the latter updates it.

Reply:
Will do.